src/us/kbase/shock/client/test/ShockTests.java

```diff
@@ -669,36 +679,22 @@ private void testFile(String content, String name, String format, ShockNode sn)
 		assertThat("file content unequal", filecon1, is(content));
 		assertThat("file name unequal", snget.getFileInformation().getName(),
 				is(name));
-		assertThat("file format unequal", snget.getFileInformation().getFormat(),
+		assertThat("file format unequal", snget.getFormat(),
 				is(format));
 		assertThat("file size wrong", snget.getFileInformation().getSize(),
 				is((long) content.length()));
 	}
 	
 	@Test
 	public void invalidFileRequest() throws Exception {
-		ShockNode sn = BSC1.addNode(new ByteArrayInputStream("".getBytes()), 0, "f", null);
-		try {
-			sn.getFile(new ByteArrayOutputStream());
-			fail("Got file from node w/o file");
-		} catch (ShockNoFileException snfe) {
-			assertThat("no file exc string incorrect", snfe.getLocalizedMessage(), 
-					is("Node has no file"));
-		}
+		ShockNode sn = BSC1.addNode(new ByteArrayInputStream("a".getBytes()), 1, "f", null);
```

Comment:
You can't store an empty file in the blobstore so all these tests that throw exceptions on empty nodes are impossible to run